specification.md

```diff
@@ -173,7 +185,7 @@ Note: this document requires a simplification pass to reduce the scope, size and
 
 </details>
 
-# Introduction
+## Introduction
```

Comment:
@tigrannajaryan - since there is no significant content above this point, I'd drop this header. It naturally follows that the prose below the page title is introductory. E.g., there is no "Introduction" heading in the [OTLP spec](https://opentelemetry.io/docs/specs/otlp/). WDYT?